src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,136 @@
+package com.wolox.training.models;
+
+import jakarta.persistence.*;
```

Comment:
Use the specific import, to avoid bringing the whole package in a unnecessary way